src/visitor/sqlite.rs

```diff
@@ -118,7 +118,7 @@ impl Bindable for ParameterizedValue {
     }
 }
 
-// TODO: This most likely needs to be in another class.
+// TODO: This most likely should be in another class, as it is not related to sqlite.
```

Comment:
Um... it is kind of related to sqlite...

Reply:
My bad, looked in the wrong namespace.